test_suite/tests/expand/derive/enum/de_enum.expanded.rs

```diff
@@ -0,0 +1,1373 @@
+                                        }
+                                    } {
+                                        _serde::export::Some(__value) => __value,
+                                        _serde::export::None => {
+                                            return _serde::export::Err(
+                                                _serde::de::Error::invalid_length(
+                                                    0usize,
+                                                    &"struct variant DeEnum::_Map2 with 4 elements",
+                                                ),
+                                            );
+                                        }
+                                    };
+                                    let __field1 = match match _serde::de::SeqAccess::next_element::<
+                                        B,
+                                    >(
+                                        &mut __seq
+                                    ) {
+                                        _serde::export::Ok(__val) => __val,
+                                        _serde::export::Err(__err) => {
+                                            return _serde::export::Err(__err);
+                                        }
+                                    } {
+                                        _serde::export::Some(__value) => __value,
+                                        _serde::export::None => {
+                                            return _serde::export::Err(
+                                                _serde::de::Error::invalid_length(
+                                                    1usize,
+                                                    &"struct variant DeEnum::_Map2 with 4 elements",
+                                                ),
+                                            );
+                                        }
+                                    };
+                                    let __field2 = match match _serde::de::SeqAccess::next_element::<
+                                        C,
+                                    >(
+                                        &mut __seq
+                                    ) {
+                                        _serde::export::Ok(__val) => __val,
+                                        _serde::export::Err(__err) => {
+                                            return _serde::export::Err(__err);
+                                        }
+                                    } {
+                                        _serde::export::Some(__value) => __value,
+                                        _serde::export::None => {
+                                            return _serde::export::Err(
+                                                _serde::de::Error::invalid_length(
+                                              
```

Comment:
Could you make sure these files are written as proper text files with a newline character at the end of every line?

Reply:
This is an issue that require modification to `macrotest`, I filed it here: https://github.com/eupn/macrotest/issues/33

Reply:
Done, updated the crate.